code/game/objects/items/weapons/landmines.dm

```diff
@@ -171,10 +243,164 @@
 
 	qdel(src)
 
+/**
+ * # EMP Landmine
+ *
+ * A landmine that emits an EMP pulse
+ */
 /obj/item/landmine/emp
 	icon_state = "emplandmine"
 
-/obj/item/landmine/emp/trigger(mob/living/L)
+/obj/item/landmine/emp/trigger(mob/living/triggerer)
 	spark(src, 3, alldirs)
 	empulse(src.loc, 2, 4)
 	qdel(src)
+
+
+/**
+ * # Phoron Landmine
+ *
+ * A landmine that only explodes when pressure is released
+ */
+/obj/item/landmine/standstill
+	name = "Standstill Landmine"
+	desc = "A landmine that only triggers if you release the pressure from the trigger."
+	icon_state = "standstill"
+	var/engaged_by = null
+
+/obj/item/landmine/standstill/Destroy()
+	STOP_PROCESSING(SSfast_process, src)
+	engaged_by = null
+	. = ..()
+
+/obj/item/landmine/standstill/trigger(mob/living/triggerer)
+	if(!engaged_by && !deactivated)
+		to_chat(triggerer, SPAN_HIGHDANGER("Something clicks below your feet, a sense of dread permeates your skin, better not move..."))
+		engaged_by = ref(triggerer)
+
+		//Because mobs can bump and swap with one another, and we use forcemove that doesn't call Crossed/Uncrossed/Entered/Exited, we have to
+		//keep looking if the victim is still on the mine, and otherwise explode the mine
+		START_PROCESSING(SSfast_process, src)
+		INVOKE_ASYNC(GLOBAL_PROC, GLOBAL_PROC_REF(tgui_alert), triggerer, "You get a dreadful feeling from your [pick("right", "left")] foot...", "Dread", list("Mom..."))
```

Comment:
this message isn't very clear, I'm not sure what it's supposed to convey

Reply:
It's supposed to convey that you feel something very bad (pressing down the trigger of the mine) under your foot, how would you reword it to make it more clear without it being an obvious "AYO YOU STEPPED ON A MINE!" ?

Reply:
Just do `You feel your [left/right] foot step down on a button with a click...`, `Uh...`, `Oh...`